ecomap/ecomap/Model/EcomapStatsParser.m

```diff
@@ -0,0 +1,95 @@
+//
+//  EcomapStatsParser.m
+//  ecomap
+//
+//  Created by ohuratc on 09.02.15.
+//  Copyright (c) 2015 SoftServe. All rights reserved.
+//
+
+#import "EcomapStatsParser.h"
+#import "EcomapPathDefine.h"
+
+@implementation EcomapStatsParser
+
++ (id)valueForKey:(NSString *)key inGeneralStatsArray:(NSArray *)generalStats
+{
+    for(id stats in generalStats) {
+        if([stats isKindOfClass:[NSArray class]]) {
+            if([[(NSArray *)stats firstObject] isKindOfClass:[NSDictionary class]]) {
```

Comment:
It's because too complicated logic of returning JSON from Ecomap server. It returns JSON with array which has three arrays. Every array has one element which is dictionary with different amount of entries.